tests/acceptance/pages.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import json
+
+
+class BasePage(object):
```

Comment:
I've cribbed this idea from #12632 and will share implementation later when both pulls have merged.